tests/test_router_quote.py

```diff
@@ -0,0 +1,244 @@
+import time
+import pytest
+
+import conftest
+from sqs_service import *
+from quote_response import *
+from rand_util import *
+from e2e_math import *
+from decimal import *
+from constants import *
+from util import *
+
+SQS_STAGE = "https://sqs.stage.osmosis.zone"
```

Comment:
Looks like we have multiple places in tests suite pointing to https://sqs.stage.osmosis.zone, /router/quotes and /router/routes. Shall we move them to constants.py? Same for QUOTE_NUM_TOP_LIQUIDITY_DENOMS, TRANSMUTER_MIN_TOKEN_LIQ_USD, expected_latency_upper_bound_ms, like what we did in test_tokens_prices.pay. Hopefully we don't have cyclic dependency issue and if so, it's okay to make them stay here.

Reply:
Removed duplicate stage - was an erroneous copy. Thanks.

Kept `TRANSMUTER_MIN_TOKEN_LIQ_USD` in `constants.py` since it is used across many test components.

On the other hand, `QUOTE_NUM_TOP_LIQUIDITY_DENOMS` is here for a reason. Instead of creating a bloated `constants.py` with identifiers that are only used in individual files, I prefer to define them in test files where they are used directly.

`constants.py` instead should contain constants that are shared across multiple files or components.

Reply:
Make sense. Thannks!